src/plays/pokemon-stats/PokemonStats.js

```diff
@@ -0,0 +1,40 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import { useState } from "react";
+import "./styles.css";
+import axios from "axios";
+import Pokemoncard from "./components/Pokemoncard";
+import Search from "./components/Search";
+// WARNING: Do not change the entry componenet name
+function PokemonStats(props) {
+  // Your Code Start below.
+  const [pokemon, setpokemon] = useState();
+  const [name, setname] = useState("");
+  const Pokemon = (e) => {
```

Comment:
This method name can be like fetchPokemonDetails. The convention used look like we are creating a class.